src/display.c

```diff
@@ -139,16 +139,22 @@ plot_print_y_label(struct plot *p, struct canvas_elem e, double l, int side)
 
 	pp = side == 1 ? PPTLeft | ((e.peice & 0x8) >> 2) : PPTRight | e.peice;
 
+	if (side == 1) {
+		printf("\033[0m");
```

Comment:
You should only print this out `if (p->color)` so that no escape sequences are generated if no colored output is enabled

Reply:
Acknowledged, done in the next patchset.